Core/Core/Kits/Applications.cs

```diff
@@ -2,7 +2,138 @@
 
 namespace Speckle.Core.Kits
 {
-  [Obsolete("Use VersionedHostApplications or HostApplications instead.")]
```

Comment:
Was this `Obsolete` attribute intended to be deleted? It seems like the `Obsolete` attrs were added bellow too, but better be sure :)